WORKSPACE

```diff
@@ -34,11 +39,9 @@ http_archive(
 
 http_archive(
     name = "com_github_google_glog",
-    # Using the same version as in kythe
-    # https://github.com/kythe/kythe/blob/410f69c5bcb69fabcb78a5200b7631a1bffabd31/external.bzl#L155
-    sha256 = "9b4867ab66c33c41e2672b5de7e3133d38411cdb75eeb0d2b72c88bb10375c71",
-    strip_prefix = "glog-ba8a9f6952d04d1403b97df24e6836227751454e",
-    urls = ["https://github.com/google/glog/archive/ba8a9f6952d04d1403b97df24e6836227751454e.zip"],
+    sha256 = "62efeb57ff70db9ea2129a16d0f908941e355d09d6d83c9f7b18557c0a7ab59e",
```

Comment:
Good. With Kythe out of the way, we can upgrade.

I suggest to send this glog update as separate pull request.

Reply:
Maybe we can use the almost-next-release version https://github.com/google/glog/archive/refs/tags/v0.5.0-rc2.zip

That way, it provides us a visual hint that we should update to v0.5.0 once it is out.

Reply:
Have you considered using renovatebot to keep dependencies up-to-date?

Reply:
Only partially. Reason is, that almost always when there is a dependency changed, it turns out it requires some human intervention, so I like to do it as a deliberate action every now and then (example here: the particular version was chosen because of implicit Kythe dependency).

Reply:
> Where are we using designated initializers ? I have not found an option in gcc or clang to warn about designated initializers so couldn't find all the places easily.
> 
> Maybe we can just comment uses of `.foo = bar` as `/* .foo = */ bar` for now until we're on C++20.

There is [verilog_linter_configuration_test](https://github.com/google/verible/blob/713e510250ba0f18e3fd634e27c7c4a182647411/verilog/analysis/verilog_linter_configuration_test.cc#L782), [kythe_facts_extractor](https://github.com/google/verible/blob/713e510250ba0f18e3fd634e27c7c4a182647411/verilog/tools/kythe/kythe_facts_extractor.cc#L678), [symbol_table](https://github.com/google/verible/blob/713e510250ba0f18e3fd634e27c7c4a182647411/verilog/analysis/symbol_table.cc#L448) and at least 5 other files.

There are other files that won't compile with MSVC (notably `editscript.h`). For the moment, I suggest we go with `clang-cl`. Once we have a version working on Windows, we can update the project to compile on MSVC.

Reply:
Sounds good. One step at a time.